security/s2a/internal/crypter/aesgcm_test.go

```diff
@@ -20,53 +20,11 @@ package crypter
 
 import (
 	"bytes"
-	"encoding/hex"
-	"encoding/json"
 	"fmt"
-	"os"
+	. "google.golang.org/grpc/security/s2a/internal/crypter/testutil"
 	"testing"
 )
 
-// Constants indicating whether the test vector is valid or not.
-const (
-	validResult   = "valid"
-	invalidResult = "invalid"
-)
-
-// cryptoTestVector is a struct for a AES-GCM test vector
-type cryptoTestVector struct {
-	id                                          int
-	key, plaintext, ciphertext, tag, nonce, aad []byte
-	result, comment                             string
-	allocateDst                                 bool
-}
-
-// testVector is a struct for a WycheProof test vector
-type testVector struct {
-	TcId    int    `json:"tcId"`
-	Comment string `json:"comment"`
-	Key     string `json:"key"`
-	Iv      string `json:"iv"`
-	Aad     string `json:"aad"`
-	Msg     string `json:"msg"`
-	Ct      string `json:"ct"`
-	Tag     string `json:"tag"`
-	Result  string `json:"result"`
-}
-
-// testGroup is a struct for a WycheProof test group
-type testGroup struct {
-	IvSize  int          `json:"ivSize"`
-	KeySize int          `json:"keySize"`
-	TagSize int          `json:"tagSize"`
-	Tests   []testVector `json:"tests"`
-}
-
-// testFile is a struct for a WycheProof test file
-type testFile struct {
-	TestGroups []testGroup `json:"testGroups"`
-}
-
 // getGCMCryptoPair outputs a client/server pair on AES-GCM.
```

Comment:
Can we call these something else than `client` and `server`? Perhaps `protector` and `unprotector`, or `sender` and `receiver`?

Reply:
I took the naming from the ALTS unit tests. I think the consistency of the naming between this and ALTS would be better kept the same, WYDT?

Reply:
The `client` and `server` naming in this context implies that the client does the encrypting and the server does the decrypting, which is not the case - both do both. That's why I was suggesting the different naming. I'm not sure why ALTS chose that naming scheme, but I don't think we should keep it just for the sake of consistency.

Reply:
Got it, will change it then!